keps/sig-instrumentation/3498-extending-stability/README.md

```diff
@@ -311,19 +344,18 @@ This should not affect upgrade/rollback paths.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-`Alpha` metrics will be recategorized as `Internal`.
+No.
```

Comment:
Sorry - it seems that you changed the wrong line.
Let's revert thisline (recategorizing Alpha metrics is a good call out here).

And instead let's change L367 to "no"

`###### Are there any missing metrics that would be useful to have to improve observability of this feature?`

Reply:
Sure